.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+      - edited
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  markdownlint:
+    name: Markdown Linter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout EIP Repository
+        uses: actions/checkout@47fbe2df0ad0e27efb67a70beac3555f192b062f
+
+      - name: Get Changed Files
+        id: changed-files
+        run: |
+          CHANGED_FILES=$(gh pr diff ${{ github.event.pull_request.number }} --name-only | grep -E '\.md$' || true)
+          echo "files=$CHANGED_FILES" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Lint
+        uses: DavidAnson/markdownlint-cli2-action@f5cf187ef11bd3a68a127321b794aa252ff23019
+        if: steps.changed-files.outputs.files != ''
+        with:
+          config: config/.markdownlint.yaml
+          globs: ${{ steps.changed-files.outputs.files }}
```

Comment:
please add a new line at the bottom to remove 
![image](https://github.com/user-attachments/assets/f479cbd5-625d-4c83-b9f3-bfb956197a30)